build.env

```diff
@@ -31,6 +31,7 @@ HELM_VERSION=v3.1.2
 
 # minikube settings
 MINIKUBE_VERSION=v1.12.1
+KUBE_VERSION=v1.18.8
```

Comment:
Adding a variable is fine, but the commit message should describe why it is used. You can also add the actual usage in the same commit to make things clearer. No need to split everything in very small commits.

Reply:
If we do not want to source the build.env in the minikube.sh, then we don't need this change.
Thanks!